galactic/distribution.yaml

```diff
@@ -2631,6 +2631,24 @@ repositories:
       url: https://github.com/ros-perception/perception_pcl.git
       version: foxy-devel
     status: maintained
+  performance_test-release:
```

Comment:
@revati-naik this is unusual, the entry name typically matches the source repository name, not the release's. I suspect `bloom` got confused as well -- note the `release repository: unknown` entries in https://github.com/ros2-gbp/performance_test-release README.md.

Reply:
@hidmic Thank you for pointing this out. I have pushed the required changes.